js/client-teambuilder.js

```diff
@@ -1289,8 +1289,12 @@
 					success: function (data) {
 						if (/^https?:\/\/pokepast\.es\/.*\/json\s*$/.test(url)) {
 							var teamData = JSON.parse(data);
-							var format = window.BattleFormats && window.BattleFormats[toID(teamData.notes.slice(8))];
-							if (format) self.changeFormat(format.id);
+							var notes = teamData.notes;
+							if (notes.startsWith('Format: ')) {
+								var formatid = toID(notes.slice(8, notes.indexOf('\n')));
```

Comment:
I think this breaks if `notes` doesn't contain `\n`.

Reply:
i fixed that in my latest commit